### ISSUE TITLE
Can't use FR dict with Kobo Clara HD 4.38

### ISSUE THREAD

**Issue by slatrille-vl** [BoboTiG/ebook-reader-dict#2200]
Hello,

First thing first, thanks for the work you're doing!

I have an issue with the FR dictionnary on a Kobo Clara HD running the 4.38 firmware. I can install and select the dict with no problem but can't find any definition when looking for one while reading. Result is always empty.

I tried to install the dict ZIP in .kobo/dict or .kobo/custom-dict with a reboot and a verification that dict is present and selected in Kobo dictionnary parameters but nothing worked.

Do you have an idea?

Best regards,

Stephan

**Comment by BoboTiG** [comment]
And does it work with the EN dict, or any other locale?

**Comment by slatrille-vl** [comment]
I tried both FR and EN dictionnaries on another Kobo Clara HD running the 4.25 firmware. I installed the dicts in custom-dist then dict rebooting each time and verifying that dictionnaries were loaded and selected in parameters. 

I didn't find a definition in french (french book) or in english (english book). I tried with simple words and verified that Kobo official dictionnary was still working well. I even tried to switch the Kobo language with no result.

Don't understand what I am doing wrong...

**Comment by BoboTiG** [comment]
So it does not work on firmware 4.25, correct?

I just tried again with a fresh FR dict. I selected a word, then I had to select the "Français - Française (Personnalisé)" dictionary: the definition showed up!

I have a Kobo Libra H2O, I can't test on other devices.

**Comment by lasconic** [comment]
It works as expected for me. Kobo Aura One, firmware 4.38.23038.
The `dicthml-fr-fr.zip` is in `.kobo/custom-dict`.

I could try on a Clara HD in the coming days if necessary.

**Comment by slatrille-vl** [comment]
> It works as expected for me. Kobo Aura One, firmware 4.38.23038. The `dicthml-fr-fr.zip` is in `.kobo/custom-dict`.
> 
> I could try on a Clara HD in the coming days if necessary.

yes please

**Comment by slatrille-vl** [comment]
> So it does not work on firmware 4.25, correct?
> 
> I just tried again with a fresh FR dict. I selected a word, then I had to select the _Français - Français (Personnalisé)_ dictionary: the definition showed up!
> 
> I have a Kobo Libra H2O, I can't test on other devices.
> 
> One thing you could try is to rename `dicthml-fr-fr.zip` to `dicthtml-fr.zip` and move the file into the `.kobo/dict` folder (and not `.kobo/custom-dict`).

Yes, same thnig on 4.25. I tried to replace the FR standard dict by moving your dict in .kobo/dic and naming it dicthtml-fr.zip but had the same results, empty search. Is there a way to get logs on the kobo reader ?

**Comment by BoboTiG** [comment]
> Is there a way to get logs on the kobo reader ?

Yes, using [NickelMenu](https://github.com/pgaskin/NickelMenu/). You can find the configuration I use [here](https://www.tiger-222.fr/luma/diy/liseuse-kobo-libra-h2o.html#nickelmenu).

**Comment by BoboTiG** [comment]
@lasconic I am curious to know if you have issues too on you Clara :thinking:

**Comment by lasconic** [comment]
tl;dr : it works on my Kobo Clara HD firmware 4.38


Sorry for the delay.
Kobo Clara HD firmware 4.38.23038
* Download the full KOBO french dictionary from https://github.com/BoboTiG/ebook-reader-dict/releases/tag/fr
* Plug the kobo clara hd
* Copy the zip in `.kobo/custom-dict`
* Unplug the device
* Open a book in french, long press on a word
* Select "Français - Français (Personnalisé)"
* The definition is displayed

**Comment by BoboTiG** [comment]
I close the icket since it seems at target issue on your device @slatrille-vl. Maybe could you find a better support on mobileread forums?
